system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java

```diff
@@ -48,4 +62,27 @@ void listParticipants() {
         assertThat(api.listParticipants())
                 .anySatisfy(p -> assertThat(p.getUrl()).isEqualTo(participantUrl));
     }
+
+    @Test
+    void addsVerifiableCredential() {
+        // sanity check
+        assertThat(getVerifiableCredentialsFromIdentityHub()).noneSatisfy(this::assertIssuedVerifiableCredential);
+
+        api.addParticipant(participantUrl);
+
+        await().atMost(2, MINUTES).untilAsserted(() -> {
+            assertThat(getVerifiableCredentialsFromIdentityHub()).anySatisfy(this::assertIssuedVerifiableCredential);
+        });
+    }
+
+    private Collection<SignedJWT> getVerifiableCredentialsFromIdentityHub() {
+        var result = identityHubClient.getVerifiableCredentials("http://localhost:8181/api/identity-hub");
+        assertThat(result.succeeded()).isTrue();
+        return result.getContent();
+    }
+
+    private void assertIssuedVerifiableCredential(SignedJWT jwt) throws ParseException {
+        assertThat(jwt.getJWTClaimsSet().getSubject()).isEqualTo(CLIENT_DID_WEB);
```

Comment:
Verify "memberOfDataspace" claim?

Reply:
Intentionally not done - here we do minimal assertions to ensure a VC was pushed (the assertions are to ensure it is not some older VC). The claim is covered in UT.